### PR TITLE
Fix custom logger bug

### PR DESCRIPTION
**Pull request by YahyaGrb**
**Proposed changes**:
- the new custom logging feature is not working due to small syntax issue in the argparse level. The file name is never retrieved so it will never create the new custom config file with the desired formatting 

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation: No need
- [X] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [X] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by ancalita** [comment]
Also @YahyaGrb  please ensure to sign [the license agreement](https://github.com/RasaHQ/rasa/pull/11963#issuecomment-1402103058) 🙏🏻

File: changelog/11963.bug.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+```
```
[the remark this answers is not in the document]
Comment (YahyaGrb) [reply changelog/11963.bug.md:1]: thanks for the feedback, I could not find the 3.4.X on my fork :(. will try again

[the remark this answers is not in the document]
Comment (YahyaGrb) [reply changelog/11963.bug.md:1]: image
double checked, can't find the branch 3.4.x

[the remark this answers is not in the document]
Comment (YahyaGrb) [reply changelog/11963.bug.md:1]: changelog updated, could you please rebase it if you have enough rights? can not do it on my side.

**Comment by YahyaGrb** [comment]
I did sign it with my github (YahyaGrb), but I pushed some commits with another local git config so the User Yahya GHRAB could not sign, but should be ok, no ?

File: changelog/11963.bug.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ancalita) [reply changelog/11963.bug.md:1]: I don't think I can rebase on your fork. Have you tried this in your terminal?
```
1. `git rebase --onto 3.4.x main fix-custom-logger-bug`
2. `git push --force origin fix-custom-logger-bug`
```

[the remark this answers is not in the document]
Comment (ancalita) [reply changelog/11963.bug.md:1]: Or maybe even first `git pull` while on `main`?

[the remark this answers is not in the document]
Comment (YahyaGrb) [reply changelog/11963.bug.md:1]: would not work, it is the fork of rasa that does not retrieve `3.4.x.` I can not change the fork , I sync it with the latest version but `3.4.x` is never retrieved

[the remark this answers is not in the document]
Comment (YahyaGrb) [reply changelog/11963.bug.md:1]: tried all the above. no branch 3.4.x on the fork. I think your org protects the branch 3.4.x from being pulled by fork

[the remark this answers is not in the document]
Comment (ancalita) [reply changelog/11963.bug.md:1]: Hey @YahyaGrb I have checked with our InfraSec team, and there's no extra specific security controls implemented. I would recommend re-forking and unticking the `copy main branch only` box, it's possible you forked with this default option ticked which prevents you from getting all branches.
image

**Pull request closed without merging**